src/views/forms/TradeForm/TradeFormInputs.tsx

```diff
@@ -86,7 +86,7 @@ export const TradeFormInputs = () => {
           <WithTooltip tooltip="trigger-price" side="right">
             {stringGetter({ key: STRING_KEYS.TRIGGER_PRICE })}
           </WithTooltip>
-          <Tag>USD</Tag>
+          <Tag tw="ml-0.25">USD</Tag>
```

Comment:
oh this was my bad previously - should be fixed now 😢 I accidentally removed margin from `WithLabel` - I don't think we need the margin here anymore